examples/namespaced-types/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>vuex namespaced types example</title>
+    <link href="http://fonts.googleapis.com/css?family=Muli" rel="stylesheet" type="text/css">
```

Comment:
Hey @ktsn - I just copied another example, but I can see it isn't used. Will remove it.